src/ReplicatedStorage/Client/InterfaceComponents/TextLabel.lua

```diff
@@ -0,0 +1,91 @@
+local ReplicatedStorage = game:GetService("ReplicatedStorage")
+local Packages = ReplicatedStorage:WaitForChild("Packages")
+local Fusion = require(Packages.Fusion)
+local WrappedFusion = require(Packages.FusionUtils).Wrapped
+
+local New = Fusion.New
+local Statify = WrappedFusion.Statify
+local Children = Fusion.Children
+local Computed = Fusion.Computed
+
+-- Project-wide defaults
+local DEFAULT_FONT = Enum.Font.GothamBlack
+local DEFAULT_TEXT = ""
+local DEFAULT_TEXT_COLOR = Color3.new(1, 1, 1)
+local DEFAULT_TEXT_POSITION = UDim2.new(0, 0, 0.15, 0)
+local DEFAULT_TEXT_TRANSPARENCY = 0
+local DEFAULT_SHADOW_TRANSPARENCY = 1
+local DEFAULT_SHADOW_OFFSET = UDim2.new(0, 0, 0.125, 0)
+local DEFAULT_SHADOW_COLOR = Color3.new()
+local DEFAULT_STROKE_SIZE = 0
+local DEFAULT_STROKE_TRANSPARENCY = 0
+local DEFAULT_STROKE_COLOR = Color3.new()
```

Comment:
Don't defaults make our UI less declarative?

Reply:
They override the Fusion defaults so I don't think that's necessarily true. You will always have defaults, we just want different defaults.